Cesium3DTiles/src/RasterOverlayTileProvider.cpp

```diff
@@ -524,6 +525,58 @@ RasterOverlayTileProvider::loadTileImageFromUrl(
           });
 }
 
+RasterOverlayTileProvider::LoadResult
+RasterOverlayTileProvider::processLoadedImage(
+    const CesiumGeometry::QuadtreeTileID& tileId,
+    LoadedRasterOverlayImage&& loadedImage) {
+
+  const auto& pPrepareRendererResources = this->getPrepareRendererResources();
+  const auto& pLogger = this->getLogger();
+
+  if (!loadedImage.image.has_value()) {
+    std::string tileIdString =
+        Cesium3DTiles::TileIdUtilities::createTileIdString(tileId);
+    SPDLOG_LOGGER_ERROR(
+        pLogger,
+        "Failed to load image for tile " + tileIdString + ":\n- {}",
```

Comment:
Minor point, but it's better to use `{}` for `tileIdString` rather than building the string with `+`. And while it's probably irrelevant for an error, it's also slightly better to put the call to `Cesium3DTiles::TileIdUtilities::createTileIdString(tileId)` right inside the `SPDLOG_LOGGER_ERROR` call rather than assigning it to a local variable beforehand. That way the call can be eliminated entirely if that log level is disabled with a compile-time flag.

Reply:
I'm aware of that. We seem to have different priorities here.

Reply:
What priority did you have here that my suggestion conflicts with?